lib/irb/command/cd.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module IRB
+  module Command
+    class CD < Base
+      category "Workspace"
+      description "Move into the given object or leave the current context."
+
+      help_message(<<~HELP)
+        Usage: cd ([target]|..)
+
+        When given an object, cd will move into that object, making it the current context.
+        When given "..", cd will leave the current context, moving back to the previous context.
+        When given no arguments, cd will move back to the top-level (main object) context.
+
+        Examples:
+
+          cd Foo
+          cd Foo.new
+          cd @ivar
+          cd ..
+          cd
+      HELP
+
+      def execute(arg)
+        case arg
```

Comment:
How about the `cd -` case? As in, go to the previous but also push the current back on the stack.